### DISCUSSION TITLE
Postgres: Make raw query text area expandable

### DISCUSSION THREAD

**Discussion by mikkokotila** [grafana/grafana#38330] [Data Sources]
**What would you like to be added**:

Very small changes can radically change the develop experience with Granfana. This means the depth it already has, becomes much more available in very short period of time simply because attention goes to less risky, faster to implement, experience focused aspects. 

For example, current visual shows where someone like myself spends great majority of the time in Grafana. That's the real-estate situation. There is no way to make the query window larger. There is no apparent way to increase the font size from the GUI. Browser zoom is not supported. 

![image](https://user-images.githubusercontent.com/7943188/102016329-4d67d980-3d58-11eb-8178-d1784ba4ae88.png

I think this will be a really great place to start. A lot of the answers on Grafana end up being "no we don't do that, do it on query level". Yes ok, but let's make the query experience great then.

**Comment by torkelo** [comment]
What do you mean browser zoom is not supported? Works great in every browser I test with

**Comment by torkelo** [comment]
Maybe the browser zoom affects everything but not the query code editor?

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gabor** [comment]
hi @mikkokotila , in the current grafan version there is an "Expand Editor" button at the bottom-right corner of the text editor, that will make the editor as large as possible. i think this should solve your problem.

if that's not the case, please create a new github issue and describe how this solution is not sufficient. thanks!
